src/modules/feature-limit/limits.enum.ts

```diff
@@ -0,0 +1,9 @@
+export enum LimitLabels {
+  MaxSharedItems = 'max-shared-items',
+  MaxSharedItemInvites = 'max-shared-invites',
+}
+
+export enum LimitTypes {
+  Boolean = 'boolean',
+  counter = 'counter',
```

Comment:
counter should be Counter as key, following the TypeScript official conventions, like you have done with Boolean, which has the first letter in uppercase.